src/main/jbake/content/jsonp005.adoc

```diff
@@ -11,24 +11,24 @@ prev=jsonp004.html
 JSON in Jakarta EE RESTful Web Services
 ---------------------------------------
 
-This section explains how the Java API for JSON Processing is related to
+This section explains how the Jakartaa JSON Processing is related to
 other Jakarta EE packages that provide JSON support for RESTful web
 services. See link:jaxrs.html#GIEPU[Chapter 32, "Building RESTful Web
-Services with JAX-RS,"] for more information on RESTful web services.
+Services with Jakarta RESTful Web Services,"] for more information on RESTful web services.
 
-Jersey, the reference implementation for JAX-RS (JSR 311) included in
+Jersey, the Jakarta RESTful Web Services implementation included in
 GlassFish Server, provides support for binding JSON data from RESTful
-resource methods to Java objects using JAXB, as described in
-link:jaxrs-advanced007.html#GKKNJ[Using JAX-RS with JAXB] in
-link:jaxrs-advanced.html#GJJXE[Chapter 34, "JAX-RS: Advanced Topics and
-an Example"]. However, JSON support is not part of JAX-RS (JSR 311) or
-JAXB (JSR 222), so that procedure may not work for Jakarta EE
+resource methods to Java objects using Jakarta XML Binding, as described in
+link:jaxrs-advanced007.html#GKKNJ[Using Jakarta RESTful Web Services with Jakarta XML Binding] in
+link:jaxrs-advanced.html#GJJXE[Chapter 34, "Jakarta RESTful Web Services: Advanced Topics and
+an Example"]. However, JSON support is not part of Jakarta RESTful Web Services or
+Jakarta XML Binding, so that procedure may not work for Jakarta EE
 implementations other than GlassFish Server.
 
-The Java API for JSON Processing (JSR 374) does not explicitly support
+Jakarta JSON Processing does not explicitly support
 JSON binding in Java. A future JSR (JSON Binding) that is similar to
-JAXB for XML is under consideration for a future release of Jakarta EE.
+Jakarta JSON Binding for XML is under consideration for a future release of Jakarta EE.
```

Comment:
This is out of date since JSON Binding is part of Jakarta EE 8.
This paragraph should probably be removed, but I think there's other text in this section that
needs to be updated to recognize that JSON Binding is included.

Reply:
I saw this paragraph but I kept it in an attempt to modify the mininum stuff, but you're right. I've just removed the paragraph for now. There's already a section about Jakarta JSON Binding so I think we'll have time to add some more text clarifying it.